.clangd

```diff
@@ -8,3 +8,7 @@ CompileFlags:
     - -std=c11
     - -Ideps/sokol
     - -Ideps/cimgui
+    - -Invapi
+    - -D_COSMO_SOURCE
```

Comment:
This is manually defined per file, only for cosmo related source.

Reply:
I will remove this.